application/src/main/resources/ecc.yml

```diff
@@ -32,8 +32,12 @@ connection:
     ## Connection is tried every five second until either the timeout time passes or the connection is successful.
     ##
     timeout:
-      time: 0
+      time: 60
       unit: seconds
+    retryPolicy:
```

Comment:
Some kind of validation for the settings would probably be in place. As of now (considering the code), what would the setting maxDelay of 30 bring, when using the other defalut of 5/5?

Reply:
It was done in the calculation of the intervals, if the calculated delay is greater than maxDelay, the maxDelay will be used instead.